util/llpcInternal.cpp

```diff
@@ -77,20 +77,27 @@ Function* GetEntryPoint(
 }
 
 // =====================================================================================================================
-// Emits a LLVM function call (inserted before the specified instruction), builds it automically based on return type
-// and its parameters.
-CallInst* EmitCall(
-    StringRef                     funcName,         // Name string of the function
-    Type*                         pRetTy,           // [in] Return type
-    ArrayRef<Value *>             args,             // [in] Parameters
-    ArrayRef<Attribute::AttrKind> attribs,          // Attributes
-    Instruction*                  pInsertPos)       // [in] Where to insert this call
+/// Emits a LLVM function call using the given builder. The callee is built automically based on return
```

Comment:
Please use existing comment style.

Reply:
PAL uses this comment style in a few places, so I think it is okay. It is preferable because it avoids having to re-align comments on changes.

Reply:
PAL often uses this style for stable interface files and utility container files. So I think for component  internal files, we do not need doxygen style. For interface, such as llpc.h, llpcBuilder.h as the interface boundary, we can adopt this.

Reply:
Also,  in PAL, you can find doxygen style comments often stay in interface header files to describe virtual base class methods. They are interfaces used by client drivers. So the documentation is preferable.